app/recipes/routes.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+import html
+import math
+import re
+from app import mongo
+from bson.objectid import ObjectId
+from datetime import datetime
+from flask import Blueprint, current_app, render_template, redirect, request, url_for, flash, session, Markup
+from slugify import slugify
+
+
+# --------------------- #
+#    Flask Blueprint    #
+# --------------------- #
+recipes = Blueprint("recipes", __name__)
+
+
+# -------------------- #
+#    DB Collections    #
+# -------------------- #
+allergens_collection = mongo.db.allergens
+desserts_collection = mongo.db.desserts
+measurements_collection = mongo.db.measurements
+recipes_collection = mongo.db.recipes
+users_collection = mongo.db.users
+
+
+# ------------------- #
+#    Global Helper    #
+# ------------------- #
+@recipes.context_processor
+def desserts_total():
+    desserts_count = recipes_collection.count
+    return dict(desserts_count=desserts_count)
+
+
+# ---------------------- #
+#    Helper Functions    #

```

Comment:
Let's have our `DB Collections` and `Helper Functions` definitions in a utility file and import what we need from there so we won't have to repeat code and routes.py files can focus on what they are created for - routes